frontend/pendingTranslations.ftl

```diff
@@ -208,3 +208,7 @@ whatsnew-feature-forward-some-heading = Promotional email blocking
 # length as the English version.
 whatsnew-feature-forward-some-snippet = { -brand-name-relay-premium } allows you to block only promotional emails...
 whatsnew-feature-forward-some-description = { -brand-name-relay-premium } allows you to block only promotional emails sent to an alias. You’ll receive emails like receipts but not marketing emails.
+
+whatsnew-feature-alias-to-mask-heading = Aliases are now masks
+whatsnew-feature-alias-to-mask-snippet = Notice a change? We’re now calling aliases “masks” to make { -brand-name-firefox-relay }...
```

Comment:
Was thinking instead of adding a snippet string, we could just truncate the longer description with the `text-overflow` [property](https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow), or use the `substring()` [method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/substring).

Reply:
Definitely something that would be interesting to investigate! I think `substring` would be problematic in different locales, where "characters" don't quite correspond to "letters" (e.g. Japanese characters might get split in half and suddenly look entirely different, or a Japanese snippet might be truncated way earlier than an English one), but I'm not sure about that - so that would be something to verify if you're open to looking into this.

So yeah, if you're up for looking into that and verifying it in different locales, that could definitely be a nice improvement.

Reply:
Oh, and one thing I got back last time when I submitted these to the l10n repo: there's a single character that represents an ellipsis instead of three separate dots :

```suggestion
whatsnew-feature-alias-to-mask-snippet = Notice a change? We’re now calling aliases “masks” to make { -brand-name-firefox-relay }…
```